go.mod

```diff
@@ -3,25 +3,26 @@ module github.com/appuio/seiso
 go 1.16
 
 require (
-	github.com/go-logr/logr v0.3.0 // indirect
-	github.com/googleapis/gnostic v0.5.1 // indirect
+	github.com/etdub/goparsetime v0.0.0-20160315173935-ea17b0ac3318 // indirect
 	github.com/hashicorp/go-version v1.3.0
-	github.com/imdario/mergo v0.3.10 // indirect
-	github.com/karrick/tparse v2.8.2+incompatible
+	github.com/karrick/tparse v2.4.2+incompatible
```

Comment:
Why the downgrade?

Reply:
I guess it was due to missing new-style import, i.e. in `import github.com/karrick/tparse/v2`, the `/v2` part was missing.